defaults/main.yml

```diff
@@ -249,3 +249,28 @@ atom_drmc_mock: "no"
 #   Binder 0.8: "yes" (assumes ES running on same server)
 #   Binder 0.9: "no"
 atom_drmc_es_scripts: "no"
+
+#
+# Systemd AtoM worker parameters
+#
+
+# Check templates/lib/systemd/system/atom-worker.service
+
+atom_worker_systemd_start_limit_interval_sec: "24h"
+atom_worker_systemd_start_limit_burst: "3"
+atom_worker_systemd_execstart_php_extra_args: ""
+atom_worker_systemd_execstart_worker_extra_args: ""
+atom_worker_systemd_restart: "on-failure"
+atom_worker_systemd_restart_sec: "30"
+
+atom_worker_systemd_memory_limit: "-1"
+atom_worker_systemd_error_reporting: "E_ALL"
+
+atom_worker_systemd_restart_force_exit_status: "111"
+atom_worker_systemd_success_exit_status: "111"
+
```

Comment:
I have a question about the value assigned to `atom_worker_systemd_success_exit_status`. According to the [systemd docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html) :
```
SuccessExitStatus= 
Takes a list of exit status definitions that, when returned by the main service 
process, will be considered successful termination, in addition to the normal 
successful exit status 0 and, except for Type=oneshot, the signals 
SIGHUP, SIGINT, SIGTERM, and SIGPIPE
```
Why is `111` being considered a code for successful termination? In the preceding line, 111 is added to `RestartForceExitStatus=` (i.e., a 111 exit code will cause a service restart), wouldn't it be a contradiction if it is also considered a successful exit code?

Reply:
Hi @hakamine , the main reason to use the `SucessEXitStatus` is to don't show in the worker logs an error when restarting. It is a cosmetic change.

Please, take a look to Steve's Document (Section - Case: atom-worker hits the max-job-count value):

https://docs.google.com/document/d/1sm4gABvoqRouzoUX3CBmRTy8Vg9WwU78XAY00aZ9reY/edit#

When not using `SuccessExitStatus=111`, the logs shows when reaching the `max-job-count` value:

```
Oct 07 03:42:39 vagrant php7.4[4866]: 2021-10-06 20:42:39 > Job worker stopped.
Oct 07 03:42:39 vagrant systemd[1]: atom-worker.service: Main process exited, code=exited, status=111/n/a
Oct 07 03:42:39 vagrant systemd[1]: atom-worker.service: Failed with result 'exit-code'.
Oct 07 03:42:44 vagrant systemd[1]: atom-worker.service: Scheduled restart job, restart counter is at 1.
```

But when using `SuccessExitStatus=111`:

```
Oct 07 03:45:58 vagrant php7.4[5083]: 2021-10-06 20:45:58 > Job worker stopped.
Oct 07 03:45:58 vagrant systemd[1]: atom-worker.service: Succeeded.
Oct 07 03:46:03 vagrant systemd[1]: atom-worker.service: Scheduled restart job, restart counter is at 1.
Oct 07 03:46:03 vagrant systemd[1]: Stopped AtoM worker.
Oct 07 03:46:03 vagrant systemd[1]: Started AtoM worker.
```